src/common/SloppyCRCMap.h

```diff
@@ -5,7 +5,10 @@
 #define CEPH_COMMON_SLOPPYCRCMAP_H
 
 #include "include/encoding.h"
-#include "common/Formatter.h"
+
+namespace ceph {
+class Formatter;
```

Comment:
should we kill the ceph namespace?  this seemed like a good idea at the time but it seems to be mostly an ongoing headache.

Reply:
probably, but `Formatter` is a quite a general name. i am afraid that removing `ceph` would lead to name confliction in future.